themes/aframe/source/js/docs.js

```diff
@@ -1,11 +1,15 @@
 /* global $ */
-
 (function () {
 
 var anchorHeadingsSelector = 'h2[id], h3[id], h4[id], h5[id], h6[id]';
 
 var content = $('.content');
+
 if (content) {
+  if (window.location.hash) {
+    window.location.hash = window.location.hash.toLowerCase();
```

Comment:
yep, this looks good. personally, I'd check the value first:

```js
if (window.location.hash) {
  var hashLower = window.location.hash.toLowerCase();
  if (window.location.hash !== hashLower) {
    window.location.hash = hashLower;
  }
}
```

but it's unnecessary since I know browsers are smart to not change the hash target if it's the same. I don't know too much what the browser does under the hood about that though. seems harmless.